Various improvements (#567)
07th-mod/higurashi-console-arcs: Update/mio_omake_switch.txt

	if (GetGlobalFlag(GADVMode)) { OutputLine("<color=#5ec69a>詩音</color>", NULL, "<color=#5ec69a>Shion</color>", NULL, Line_ContinueAfterTyping); }
	ModPlayVoiceLS(4, 6, "ps3/s20/06/440600185", 256, TRUE);
	OutputLine(NULL, "まぁ、たくさん話に参加できたのに文句を言っちゃ、罰が当たるってもんですけどね」",

> 			NULL, "Though thankfully we had many chances to play a part, so we should not moan about it, you know.\"", Line_Normal);

Reviewer: ```suggestion
			NULL, "Though thankfully we had many chances to play a part, so we shouldn't moan about it, you know.\"", Line_Normal);
```

	if (GetGlobalFlag(GADVMode)) { OutputLine("<color=#f0953d>レナ</color>", NULL, "<color=#f0953d>Rena</color>", NULL, Line_ContinueAfterTyping); }
	ModPlayVoiceLS(4, 2, "ps3/s20/02/440200412", 256, TRUE);
	OutputLine(NULL, "「……本当にそれ、自慢じゃないね」",

> 			NULL, "\"...Indeed, it's really not a bragging.\"", Line_Normal);

Reviewer: ```suggestion
			NULL, "\"...Indeed, that really isn't bragging.\"", Line_Normal);
```

			NULL, " Rena was the main heroine when \"Higurashi\" started, right?", Line_WaitForInput);
	ModPlayVoiceLS(4, 2, "ps3/s20/02/440200418", 256, TRUE);
	OutputLine(NULL, "　間違いなくレナだったよね？　ポスターにも、ジャケットにもピンで載ったくらいだし」",

> 			NULL, " It was definitely Rena, right? That's why there're some posters and covers of just me alone.\"", Line_Normal);

Reviewer: ```suggestion
			NULL, " It was definitely Rena, right? That's why there are some posters and covers of just me alone.\"", Line_Normal);
```

	if (GetGlobalFlag(GADVMode)) { OutputLine("<color=#8676cf>羽入</color>", NULL, "<color=#8676cf>Hanyuu</color>", NULL, Line_ContinueAfterTyping); }
	ModPlayVoiceLS(4, 12, "switch/s20/12/ot_han_0050", 256, TRUE);
	OutputLine(NULL, "「って魅音、さっき老けた自分と対面したショックからもう立ち直ったのですか？」",

> 			NULL, "\"Hey Mion, have you recovered already from the shock of meeting your future self?\"", Line_Normal);

Reviewer: ```suggestion
			NULL, "\"Hey Mion, have you recovered yet from the shock of meeting your future self?\"", Line_Normal);
```

	if (GetGlobalFlag(GADVMode)) { OutputLine("<color=#ec7bb8>田村</color>", NULL, "<color=#ec7bb8>Tamura</color>", NULL, Line_ContinueAfterTyping); }
	ModPlayVoiceLS(4, 51, "switch/s20/51/ot_tam_0020", 256, TRUE);
	OutputLine(NULL, "「中の人などおらぬッッ！ 内輪の話は控え給え、角の民の巫女よ！！」",

> 			NULL, "\"There are no such person! Do refrain from speaking of confidential matters, miko of the people of horns!!\"", Line_Normal);

Reviewer: ```suggestion
			NULL, "\"There is no such person! Do refrain from speaking of confidential matters, miko of the people of horns!!\"", Line_Normal);
```

	if (GetGlobalFlag(GADVMode)) { OutputLine("<color=#85e6e0>采</color>", NULL, "<color=#85e6e0>Une</color>", NULL, Line_ContinueAfterTyping); }
	ModPlayVoiceLS(4, 50, "switch/s20/50/ot_une_0020", 256, TRUE);
	OutputLine(NULL, "「……。我の存在……影が濃ゆい、……わけがないのです」",
			NULL, "\"...My existence... shouldn't be... conspicuous.\"", Line_Normal);
	ClearMessage();

//梨花rvS20/05/ot_rika_0220.「でしょうね。こんなキャラの強烈な神様に挟まれたんじゃ、おいそれと差し出がましく首を突っ込むわけにもいかないし」
	if (GetGlobalFlag(GADVMode)) { OutputLine("<color=#6972c1>梨花</color>", NULL, "<color=#6972c1>Rika</color>", NULL, Line_ContinueAfterTyping); }
	ModPlayVoiceLS(4, 5, "switch/s20/05/ot_rika_0220", 256, TRUE);
	OutputLine(NULL, "「でしょうね。こんなキャラの強烈な神様に挟まれたんじゃ、おいそれと差し出がましく首を突っ込むわけにもいかないし」",

> 			NULL, "\"I know it. When you're standing between two striking gods like this, you can't just go sticking your nose in things.\"", Line_Normal);

Reviewer: ```suggestion
			NULL, "\"Right? When you're standing between two striking gods like this, you can't just go sticking your nose in things.\"", Line_Normal);
```

	if (GetGlobalFlag(GADVMode)) { OutputLine("<color=#945c44>富竹</color>", NULL, "<color=#945c44>Tomitake</color>", NULL, Line_ContinueAfterTyping); }
	ModPlayVoiceLS(4, 8, "ps3/s20/08/440800165", 256, TRUE);
	OutputLine(NULL, "「自分にとって大事なものを守ることに、……僕は臆病でありすぎたのかもしれない。",

> 			NULL, "\"...to confirm the truth and to protect what mattered to me.", Line_WaitForInput);

Reviewer: ```suggestion
			NULL, "\"...To confirm the truth and to protect what mattered to me.", Line_WaitForInput);
```

	if (GetGlobalFlag(GADVMode)) { OutputLine("<color=#6972c1>梨花</color>", NULL, "<color=#6972c1>Rika</color>", NULL, Line_ContinueAfterTyping); }
	ModPlayVoiceLS(4, 5, "ps3/s20/05/440500206", 256, TRUE);
	OutputLine(NULL, "「いま自分が持っている何かは独力で勝ち取ったものもあれば、誰かから与えられて得たものもあります。",

> 			NULL, "\"Some things that you have now are obtained out of your own strength, and others are given to you from someone.", Line_WaitForInput);

Reviewer: ```suggestion
			NULL, "\"Some things that you have now were obtained out of your own strength, and others were given to you by someone else.", Line_WaitForInput);
```
